CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing
+
+### How to become a GCP Project contributor
+
+## Code of Conduct
+
+As contributors and maintainers of this GCP Project Operator, we do respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities. We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality. In short, be excellent to each other.
```

Comment:
A suggestions when writing markdown docs – write each sentence as a new line. It will make doc diffs way easier to review. (We don't enforce this rule, but there are projects that do.)

Reply:
```suggestion
As contributors and maintainers of this GCP Project Operator, we do respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality. In short, be excellent to each other.
```